Plugins/Wox.Plugin.Everything/IEverythingDllLoader.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Wox.Infrastructure;
+
+namespace Wox.Plugin.Everything
+{
+    public interface IEverythingDllLoader
+    {
+        void Load(PluginInitContext context);
+    }
+
+    public class EverythingDllLoader : IEverythingDllLoader
+    {
+        
+
+        public void Load(PluginInitContext context)
```

Comment:
Pedantic of me, but if this is not needed, can we remove it

Reply:
Totally right, I started to create a DLL loader but because of too many dependencies drop that.
good catch :)